games-action/polymc/polymc-1.1.1.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake java-pkg-2 optfeature xdg
+
+HOMEPAGE="https://polymc.org/"
+DESCRIPTION="A custom, open source Minecraft launcher"
+
+if [[ ${PV} != 9999 ]]; then
+	MY_PN="PolyMC"
+
+	# Let's use the vendored tarball to avoid dealing with the submodules directly
+	SRC_URI="
+		https://github.com/PolyMC/PolyMC/releases/download/${PV}/${MY_PN}-${PV}.tar.gz -> ${P}.tar.gz
+	"
+else
+	inherit git-r3
+
+	EGIT_REPO_URI="
+		https://github.com/PolyMC/PolyMC
+		https://github.com/MultiMC/libnbtplusplus
+		https://github.com/stachenov/quazip
```

Comment:
Currently I think that's not possible. PolyMC requires a [patch made to upstream, but not yet included in a release](https://github.com/stachenov/quazip/commit/09ec1d10c6d627f895109b21728dda000cbfa7d1). That way, we would either need to depend on the user having a -9999 version of Quazip on their system, or continue bundling it until upstream makes a new release.

Reply:
Alright, lets just keep this as is than and revisit this when quazip upstream makes their next release.

Reply:
I've opened [an issue](https://github.com/stachenov/quazip/issues/144) on Quazip's Github about that. Since the patch is already merged on upstream, it would be cool if we could just use it directly :^)

Reply:
> Then again this seems -9999 specific so I don't really mind not doing the extra mile for it.

The non-9999 ebuild also uses quazip, it is bundled in the release tarball.

Maybe it is worth the effort to make a snapshot ebuild? Though I'm not sure if PolyMC will actually build with the unmodified upstream quazip currently. There is an upstream effort to remove the custom bundled quazip version [here](https://github.com/PolyMC/PolyMC/pull/92), so I'm hoping that this issue will resolve itself in the near future.